CONTRIBUTING.md

```diff
@@ -1,17 +1,17 @@
-# Contributing
+# Contributing to Clerk's documentation
 
-Thanks for being willing to contribute to our docs!
+Thanks for being willing to contribute to [Clerk's documentation](https://clerk.com/docs)! This document outlines how to effectively contribute updates and fixes to the documentation content located in this repository.
 
 ## Project setup
 
 1.  Fork and clone the repo
-2.  Run `pnpm install` to install dependencies
+2.  Run `npm install` to install dependencies
```

Comment:
At this point, it doesn't make sense to use `pnpm` in this repository. We can change this if we decide we're going to shift to `pnpm` elsewhere.